package/Dockerfile

```
@@ -15,10 +16,12 @@ RUN if [ "${ARCH}" = "amd64" ]; then ARCH=x86_64; fi && \
     curl -sfL https://github.com/derailed/k9s/releases/download/${K9S_VERSION}/k9s_Linux_${ARCH}.tar.gz | tar xvzf -
 RUN if [ "${ARCH}" = "amd64" ] || [ "${ARCH}" = "arm64" ] || [ "${ARCH}" = "s390x" ]; then \
     curl -sLf ${KUSTOMIZE_URL} | tar -xzf - && chmod +x kustomize; \
-  fi
+    fi
 
-FROM alpine:3.12
-RUN apk add -U --no-cache bash bash-completion jq
+FROM registry.suse.com/suse/sle15:15.3
+RUN zypper -n update && \
+    zypper -n install bash-completion jq && \
+    zypper -n clean -a && rm -rf /tmp/* /var/tmp/* /usr/share/doc/packages/*
```

This will ensure that no docs are installed in the process and ensures that no sparse files end up in `/var/log/` (like `/var/log/lastlog`)
```suggestion
RUN sed -i 's/.*rpm.install.excludedocs.*/rpm.install.excludedocs = yes/g' /etc/zypp/zypp.conf && \
    zypper -n update && \
    zypper -n install bash-completion jq && \
    zypper -n clean -a && rm -rf /tmp/* /var/tmp/* /usr/share/doc/packages/* /usr/share/doc/manual/* /var/log/*
```

---

@dcermak thanks! This one is nice!